application/static/js/chase_the_ace/game/GamePage.js

```diff
@@ -6,13 +6,16 @@ class GamePage extends Phaser.Scene {
     {
         // Setting up suits and cards numbers to loop over for loading.
         const suits = ["Clubs", "Spades", "Hearts", "Diamonds"];
-        const numbers = ["Ace", "2", "3", "4", "5", "6", "7", "8", "9", "10","Jack", "Queen", "King"];
+        const numbers = ["ace", "2", "3", "4", "5", "6", "7", "8", "9", "10","jack", "queen", "king"];
 
         // Loading the game buttons, background and all of the cards.
         this.load.image("casinoRoom", "../../static/images/greentable1.jpg");
         this.load.image("startButton","../../static/images/playbutton.png");
-        this.load.image("tradeButton","../../static/images/playbutton.png");
```

Comment:
Properly comment the file to start. It can be cleaned up later but explanation should already be there.